src/components/10-atoms/input-file/index.js

```diff
@@ -0,0 +1,97 @@
+import { html } from 'lit-element';
+import { classMap } from 'lit-html/directives/class-map';
+import { unsafeHTML } from 'lit-html/directives/unsafe-html';
+/* eslint-disable import/no-extraneous-dependencies */
+import '@axa-ch/icon';
+import defineOnce from '../../../utils/define-once';
+import NoShadowDOM from '../../../utils/no-shadow';
+import styles from './index.scss';
+
+const TYPE = 'file';
+class AXAInputFile extends NoShadowDOM {
+  static get tagName() {
+    return 'axa-input-file';
+  }
+
+  static get styles() {
+    return styles;
```

Comment:
@LucaMele Why don't we use the unsafe-wrapper here? If the tests are green, why do we need that wrapper at all?

Reply:
@raphaellueckl because with the use of noShadow thats the requirement. Markus said he did document it somewhere

Reply:
Weird